lib/l10n_utils/tests/test_helpers.py

```diff
@@ -11,6 +11,19 @@
 from l10n_utils import helpers
 
 
+def test_get_locale():
+    """Test that the get_locale() helper works."""
+    eq_(helpers.get_locale('pt-BR').language, 'pt')
+    eq_(helpers.get_locale('not-a-lang').language, 'en')
+
+
+def test_get_locale_hsb():
```

Comment:
That is a good point. I called it that because it's specifically for this bug. It'll likely become more generic in future when/if we need to use this again.